### DISCUSSION TITLE
How long do you need to wait after passing  GitHub benefits

### DISCUSSION THREAD

**Discussion by yuliang615** [community/community#78335] [GitHub Education]
It shows
Congrats! Your request for @yuliang615 was approved and your benefits will be available soon.
how long do i need to wait

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @yuliang615 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
